docs/src/how-tos/build_from_source.md

```diff
@@ -0,0 +1,112 @@
+---
+id: howto-build-from-source
+title: How-To Build, Test and Run OpenRefine from source code
+sidebar_label: How-To Build from source
+---
+
+ 
+You will need:
+* [OpenRefine source code](https://github.com/OpenRefine/OpenRefine)
+* [Java JDK](http://java.sun.com/javase/downloads/index.jsp)
+* [Apache Maven](https://maven.apache.org)  (OPTIONAL)
+* A Unix/Linux shell environment OR the Windows command line
+
+From the top level directory in the OpenRefine application you can build, test and run OpenRefine using the `./refine` shell script (if you are working in a *nix shell), or using the `refine.bat` script from the Windows command line. Note that the `refine.bat` on Windows only supports a subset of the functionality, supported by the `refine` shell script. The example commands below are using the `./refine` shell script, and you will need to use `refine.bat` if you are working from the Windows command line.
```

Comment:
The `*` in `*nix` is opening an italicized section here (at least according to GitHub's markdown highlighting), so it should be escaped.

Reply:
Done.